operator-common/src/test/java/io/strimzi/operator/common/model/PemAuthIdentityTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common.model;
+
+import io.fabric8.kubernetes.api.model.Secret;
+import io.fabric8.kubernetes.api.model.SecretBuilder;
+import io.strimzi.api.kafka.model.kafka.KafkaResources;
+import org.junit.jupiter.api.Test;
+
+import static io.strimzi.test.TestUtils.map;
+import static java.util.Collections.emptyMap;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class PemAuthIdentityTest {
+    public static final String NAMESPACE = "testns";
+    public static final String CLUSTER = "testcluster";
+
+    @Test
+    public void testSecretWithMissingKeyThrowsException() {
+        Secret secretWithMissingClusterOperatorKey = new SecretBuilder()
+                .withNewMetadata()
+                .withName(KafkaResources.secretName(CLUSTER))
+                .withNamespace(NAMESPACE)
+                .endMetadata()
```

Comment:
If you indent it by the new/end sections, it will be much more readable. Here as well as in the ones below.